Makefile

```diff
@@ -101,7 +101,7 @@ help: ## Display this help.
 ## --------------------------------------
 
 .PHONY: all
-all: $(BIN_DIR)/azure-cloud-controller-manager $(BIN_DIR)/azure-cloud-node-manager $(BIN_DIR)/azure-cloud-node-manager.exe ## Build binaries for the project.
+all: $(BIN_DIR)/azure-cloud-controller-manager $(BIN_DIR)/azure-cloud-node-manager $(BIN_DIR)/azure-cloud-node-manager.exe $(BIN_DIR)/azure-acr-credential-provider $(BIN_DIR)/azure-acr-credential-provider.exe ## Build binaries for the project.
```

Comment:
Is this change related to vendor license? If not, how about creating a separate PR for it? Maybe it needs backport as well.

Reply:
all changes should be back-ported.